wp-admin/js/widgets/media-image-widget.js

```diff
@@ -0,0 +1,174 @@
+/* eslint consistent-this: [ "error", "control" ] */
+(function( component ) {
+	'use strict';
+
+	var ImageWidgetModel, ImageWidgetControl;
+
+	// @todo Implement validate methods to force the right type.
+	// Defaults will get set via WP_Widget_Image::enqueue_admin_scripts().
+	ImageWidgetModel = component.MediaWidgetModel.extend( {} );
+
+	ImageWidgetControl = component.MediaWidgetControl.extend( {
+
+		/**
+		 * Render preview.
+		 *
+		 * @returns {void}
+		 */
+		renderPreview: function renderPreview() {
+			var control = this, previewContainer, previewTemplate;
+			previewContainer = control.$el.find( '.media-widget-preview .rendered' );
+			previewTemplate = wp.template( 'wp-media-widget-image-preview' );
+			previewContainer.html( previewTemplate( { attachment: control.selectedAttachment.attributes } ) );
```

Comment:
Is there a reason `control.selectedAttachment.attributes` is not passed straight to the template?